charts/openshift-metering/values.yaml

```diff
@@ -528,8 +528,8 @@ presto:
         nodeSelector: {}
         resources:
           limits:
-            cpu: "2"
-            memory: 2Gi
+            cpu: "4"
```

Comment:
I see the majority of the values.yaml doesn't use quotes around the integer values

Reply:
True. Fixed.